.github/workflows/integration-tests.yml

```diff
@@ -95,6 +96,7 @@ jobs:
           end
         end
         EOF
+        bash ./tools/create_box_with_two_disks.sh $HOME/.vagrant.d/ ./bin/vagrant
```

Comment:
Can this be placed into the test that requires it itself rather than as part of running for all tests? Or is there something makes it difficult to run there?

Spotted that it's also referenced there so I'm wondering if this can just be removed?

Reply:
I had a problem when I want create box. If I am calling script from the runtest.sh, I have error. I don't know why i have this error, I think it's due to `sg libvirt`. It is alone solution I found.
PS: What is `sg`?

Reply:
`sg` allows executing a command as a different group id. Since in this case libvirt is being installed as part of the CI run, and as I want to ensure the tests run as a normal user to be sure they work, and therefore not run this command as root, it's necessary to ensure the user is in the correct group. However at this point I don't believe the shell used by the CI agent will have updated the user groups as that typically requires a login/logout, so to avoid that, I can just switch to explicitly using the group using sg and since the user is a member of the group, it behaves the same as though I've logged in.

Not entirely sure why it would fail, as any lack of permissions around creating file/directories as the user should cause it to fail.

Reply:
I upload box in vagrant cloud for the test : yamatoRT/tinycore-two-disks
It's simplify the code. I let the script for create box if need.